roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Get Ansible Broker project name
+  shell: oc get projects | grep -e ansible-service-broker -e automation-service-broker | awk '{print $1}'
+  register: broker_project_name
+
+- set_fact:
+    broker_project_name: '{{ broker_project_name.stdout }}'
+
+- name: Get Ansible Service Broker Configmap
+  shell: oc get configmap broker-config -o jsonpath='{.data.broker-config}' -n openshift-automation-service-broker {{ broker_project_name }}
+  register: original_configmap
+
+- set_fact:
+    original_configmap: '{{ original_configmap.stdout | from_yaml }}'
+
+- set_fact:
+    original_configmap: "{{ original_configmap|combine({'openshift': {'image_pull_policy': image_pull_policy }}, recursive=True) }}"
+
+- set_fact:
+    original_configmap: "{{ original_configmap|combine({'openshift': {'sandbox_role': sandbox_role }}, recursive=True) }}"
+
+- set_fact:
+    original_configmap: "{{ original_configmap|combine({'broker': {'launch_apb_on_bind': launch_apb_on_bind }}, recursive=True) }}"
+
+- name: Get additional broker config (from ../files)
+  set_fact:
+    additional_registry_config: "{{ lookup('file', '../files/aerogearcatalog_registry.yaml') | from_yaml }}"
+
+- name: "Append registry config to original config"
+  set_fact:
+    new_registry_config: '{{ original_configmap.registry + additional_registry_config.registry }}'
```

Comment:
is this operation idempotent? What happens when this is executed more than once?